addons.yaml

```diff
@@ -124,11 +124,14 @@ microk8s-addons:
 
     - name: "kata"
       description: "Kata Containers is a secure runtime with lightweight VMS"
-      version: "latest/stable"
-      check_status: "${SNAP_DATA}/var/lock/kata.enabled"
+      version: "3.15.0"
+      check_status: "daemonset.apps/kata-deploy"
       confinement: "classic"
       supported_architectures:
+        - arm64
         - amd64
+        - s390x
```

Comment:
Have you tested kata on all these new architectures?

Reply:
Personally no, I'm pretty limited in terms of servers I can use for experimentation, however kata-container lists it in their [supported platforms] (https://github.com/kata-containers/kata-containers?tab=readme-ov-file#platform-support).

I therefore assume that kata-containers can run on s390x (they do tests for s390x...) and I hope that _microk8s-community-addons_ test might also confirm that it can be deployed on s390s with this plugin and microk8s...